web/README.md

```diff
@@ -1,137 +1,50 @@
 # Getting started
 
-## Recommended software:
+## Required software
 
-- Node 8.12.0 or later (https://nodejs.org/en/)
-- VSCode (https://code.visualstudio.com/), or other IDE with typescript support (like WebStorm)
-- yarn package manager (https://yarnpkg.com/en/)
-- Any GIT client (standalone, or embedded into your IDE)
+-   `node` 8.12.0 or later (https://nodejs.org/en/)
+-   `yarn` package manager (https://yarnpkg.com/en/).
+    -   Please note that `npm` is not applicable for installing or building the application.
 
-## Running the project with template
-
-Clone the repository.
-
-To install react app use:
-
-```
-npm install -g create-react-app
-```
-
-To build and then start the project use:
+## Install
 
 ```
+git clone git@github.com:epam/badgerdoc.git
+cd badgerdoc
```

Comment:
![image](https://user-images.githubusercontent.com/50149163/228188412-406fbaf1-bbc0-493f-8d57-40523de12d01.png)
I guess you forgot about the `web` subfolder